plugins/notion/src/blocksToHTML.ts

```diff
@@ -94,6 +94,24 @@ export function blocksToHtml(blocks: BlockObjectResponse[]) {
             case "code":
                 htmlContent += `<pre><code class="language-${block.code.language.replace(" ", "-")}">${richTextToHTML(block.code.rich_text)}</code></pre>`
                 break
```

Comment:
We now pass the language in the dataset

```html
<pre data-language="<language>"><code>...</code></pre>
```

Reply:
I updated it to use the data-language attribute. This change required creating a mapping of Notion code languages to Framer code languages. Unsupported languages import as JavaScript since there's no way to have no code formatting on a code block.